meson.build

```diff
@@ -15,6 +15,8 @@ if get_option('test-settings')
   endif
   # Set MAX_GOP_LENGTH to 10 in tests to avoid long streams
   add_global_arguments('-DMAX_GOP_LENGTH=10', language : 'c')
+  # Option for code related to unit tests
+  add_global_arguments('-DUNIT_TEST', language : 'c')
```

Comment:
meson has changed so We'll fix this later.
Maybe change define name to SV_UNIT_TEST, in case someone else out there has the same define.